### ISSUE TITLE
Error "color-scheme"

### ISSUE THREAD

**Issue by SerhiiKhmaruk** [realh/roxterm#253]
Hello.

After the update there is an error when starting the terminal, please tell me how to solve it or how to install one of the previous versions.

(roxterm:9827): GLib-GIO-ERROR **: 12:03:42.523: Settings schema 'org.gnome.desktop.interface' does not contain a key named 'color-scheme'
Trace/breakpoint trap (core dumped)

**Comment by realh** [comment]
What distro version and GNOME version are you using? It seems like this setting was only added recently and I didn't realise. It might be possible for you to install/backport a newer version of gsettings-desktop-schemas, but that could cause other problems, so it would be safer to downgrade roxterm to 3.12 until I can fix this.

**Comment by SerhiiKhmaruk** [comment]
Hello.

serhii@work:~$ gnome-shell --version
GNOME Shell 3.36.9

serhii@work:~$ cat /etc/*-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.6 LTS"
NAME="Ubuntu"
VERSION="20.04.6 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.6 LTS"
VERSION_ID="20.04"

I aimed to install a previous version and managed to download the 3.12.1 archive. However, I'm uncertain about the installation process for this version.

**Comment by SerhiiKhmaruk** [comment]
This is the error when compiling the older version:

serhii@work:~/Downloads/roxterm-3.12.1/build$ cmake ..

> cat: version: No such file or directory
> -- Checking for modules 'gtk+-3.0>=3.22.0;dbus-1>=1.0;dbus-glib-1>=0.22'
> -- No package 'dbus-glib-1' found
> CMake Error at /usr/share/cmake-3.16/Modules/FindPkgConfig.cmake:463 (message):
>    A required package was not found
> Call Stack (most recent call first):
>   /usr/share/cmake-3.16/Modules/FindPkgConfig.cmake:643 (_pkg_check_modules_internal)
>   src/CMakeLists.txt:4 (pkg_check_modules)


> -- Configuring incomplete, errors occurred!
> See also "/home/serhii/Downloads/roxterm-3.12.1/build/CMakeFiles/CMakeOutput.log".

After opening the file "CMakeFiles/CMakeOutput.log", there is too much information and nothing is clear.

**Comment by realh** [comment]
It looks like it's missing dbus-glib-1, so in Ubuntu I think you need the `libdbus-glib-1-dev` package. There may be more though. A better way to build it is to install the `dpkg-dev` package, then int the `roxterm-3.12.1` folder (not `build`) run these commands:

- `printf 3.12.1 > version`
- `dpkg-buildpackage -b -uc -us`

You may get some errors about more missing packages in step 2, but it will tell you all the additional package names that need to be installed, so you can copy and paste them into a `sudo apt install` command, but you'll probably have to remove any bracketed version numbers. Then run the buildpackage command again.

This will create some debs in the parent directory that you can install with `sudo dpkg -i`.

**Comment by mweaver-ozette** [comment]
FTR, seeing the same issue here:
```
% roxterm
(roxterm:7582): GLib-GIO-ERROR **: 14:10:59.917: Settings schema 'org.gnome.desktop.interface' does not contain a key named 'color-scheme'
zsh: trace trap (core dumped)  roxterm
% lsb_release -ds
Ubuntu 20.04.6 LTS
```
Running Xubuntu, xfce4 version 4.14.2.

Looks like the PPA does not have older versions of roxterm, so next step is to install roxterm-3.12.1.tar.gz from source.

**Comment by mweaver-ozette** [comment]
In case it's useful to anyone else: Following @realh's instructions above, one more step so `apt-get upgrade` doesn't force you back to 3.13:
```
% sudo apt-mark hold roxterm=3.12.1-1ppa1focal1 roxterm-data=3.12.1-1ppa1focal1
```

**Comment by jima80525** [comment]
Also hit it on two machines - both running ubuntu 20.04 with Gnome shell 3.36.9.  

Is there a method for installing an 3.12.1 without building it from scratch?  A few pointers would be appreciated. :)

**Comment by realh** [comment]
This should be fixed in 3.14.2.
